joplin/js/CreateContentModal/index.js

```diff
@@ -40,14 +49,14 @@ class CreateContentModal extends Component {
     // Validate title max length
     if (this.state.titleCharacterCount > MAX_TITLE_LENGTH) return false;
 
-    // Skip Topic Select Step for creating a Department
-    if (this.state.type === 'department' && this.state.activeStep === 1) {
-      this.redirectToEditPage();
-      return;
-    }
+    // Validate title min length
+    if (this.state.titleCharacterCount <= 0) return false;
```

Comment:
We probably should alert the user when validation requirements aren't met. Otherwise they may wonder why the Continue button won't work

Reply:
Maybe also validate char count after trimming whitespace?